polygeist/tools/cgeist/Test/Verification/arrayconsllvm.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: cgeist %s --function=* -S | FileCheck %s
+// RUN: cgeist %s --function=* -S --raise-scf-to-affine=false | FileCheck %s
```

Comment:
What's the reason to deactivate the pass for this test and the other tests changed in this PR? 

Does the pass break the test?

Reply:
Yes, take this test as an example, the CHECK for the loop is no longer `scf.for`.
As these tests are not written by us, I think is best to keep the original CHECKs.

Reply:
Ok, I see. 

We might get more test-coverage for the raising pass if we also enabled it for non-SYCL tests, but that's probably something we can look into at another time.